design_system/src/main/java/org/the_chance/honymart/ui/composables/HoneyFilledButton.kt

```diff
@@ -23,6 +23,7 @@ fun HoneyFilledButton(
     label: String,
     onClick: () -> Unit,
     isEnable: Boolean = true,
```

Comment:
`isEnable` nammig could be improved for better readability and understanding of what it do.